code/dotnet/README.md

```diff
@@ -45,7 +45,7 @@ public record DatastarSignalsStore : IDatastarSignalsStore
 // add as an ASP Service
 //  allows injection of IServerSentEventGenerator, to respond to a request with a Datastar friendly ServerSentEvent
 //                  and IDatastarSignalsStore, to read what is in the data-merge-signals of the client
-builder.Services.AddDatastarGenerator<DatastarSignalsStore>();
+builder.Services.AddDatastar<DatastarSignalsStore>();
```

Comment:
Since you already have `data-merge-signals` in your code, can you go a step further and rename `DatastarSignalsStore` to `DatastarSignals`?

Reply:
I actually have a large Falco update with it changed to just ISignals. Can we merge this and I'll have that PR by end of the weekend (hopefully tomorrow)?

Reply:
Sure!